pkg/api/vlabs/validate.go

```diff
@@ -862,16 +862,6 @@ func (w *WindowsProfile) Validate(orchestratorType string) error {
 func (k *KubernetesConfig) Validate(k8sVersion string, hasWindows bool) error {
 	// number of minimum retries allowed for kubelet to post node status
 	const minKubeletRetries = 4
-	// k8s versions that have cloudprovider backoff enabled
-	var backoffEnabledVersions = common.AllKubernetesSupportedVersions
-	// at present all supported versions allow for cloudprovider backoff
```

Comment:
`at present all supported versions allow for cloudprovider backoff ` --> this validation is a no-op, remove it for now

Reply:
:)